web/models.py

```
@@ -2930,6 +2930,37 @@
+        default="pending",
+    )
+    fulfilled_by = models.ForeignKey(
+        EducationalVideo, on_delete=models.SET_NULL, related_name="fulfilling_requests", null=True, blank=True
+    )
+
+    class Meta:
+        verbose_name = "Video Request"
+        verbose_name_plural = "Video Requests"
+        ordering = ["-created_at"]
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)